src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -156,9 +160,10 @@ const DigitDisplay = (() => {
                 start        : contract.entry_tick_time,
                 end          : contract.exit_tick_time,
             };
-
-            // force rerender the table by sending the history
-            BinarySocket.send(request, { callback: redrawFromHistory });
+            if (is_redraw_possible) {
+                // force rerender the table by sending the history
+                BinarySocket.send(request, { callback: redrawFromHistory });
```

Comment:
will this cause multiple redraws on every tick received?

Reply:
Per slack talk, this won't redraw on every tick, only conditionally on contract end.